src/portalbackend/PortalBackend.DBAccess/Models/ConnectorData.cs

```diff
@@ -58,7 +60,8 @@ public record ManagedConnectorData(
     ConnectorTypeId Type,
     ConnectorStatusId Status,
     string? ProviderCompanyName,
-    Guid? SelfDescriptionDocumentId);
+    Guid? SelfDescriptionDocumentId,
+    TechnicalUserData? TechnicalUsers);
```

Comment:
please change to technicalUser

```suggestion
    TechnicalUserData? TechnicalUser);
```